docs/apis-tools/camunda-api-rest/specifications/sidebar.js

```diff
@@ -163,12 +163,6 @@ module.exports = [
         label: "Modify process instance",
         className: "api-method post",
       },
-    ],
-  },
-  {
-    type: "category",
-    label: "Process Instance",
```

Comment:
@christinaausley, I guess this reflects an inconsistent capitalization in our tags in the OpenAPI spec. This is why we have two different categories here.

Reply:
@tmetzke Can you remind me the link to the spec, and I can launch a PR for that?

Reply:
Here you go: https://github.com/camunda/camunda/blob/main/zeebe/gateway-protocol/src/main/proto/rest-api.yaml

Reply:
I have fixed this over here: https://github.com/camunda/camunda/pull/22755
Let me know if you have fixed this in a PR as well already, I couldn't find any fix for this yet 🙈

Reply:
Done with https://github.com/camunda/camunda/pull/22942/.

Reply:
This PR resolves this duplicate, and everything is now listed as items under Process instance. Were you expecting additional changes?

Reply:
No, sorry, I was just referring to the fix in the source spec and was wondering if you had created a PR for the source spec as well. I couldn't find one so I fixed this in https://github.com/camunda/camunda/pull/22755. Apparently, you had a PR open already for that which became a duplicate then, sorry for that 🙈